### PR TITLE
N2- create entities and repositories

### PR DESCRIPTION
**Pull request by dmkofgl**

### CODE REVIEW

File: src/main/java/dl/news/portal/web/config/Application.java [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @SpringBootApplication(scanBasePackages = "dl.news.portal")
+@EntityScan("dl.news.portal.domain.entity")
+@EnableJpaRepositories(basePackages = "dl.news.portal.domain.repository")
```
Comment (avinahradau) [inline src/main/java/dl/news/portal/web/config/Application.java]: Why do you need this 

> @EntityScan("dl.news.portal.domain.entity")
@EnableJpaRepositories(basePackages = "dl.news.portal.domain.repository")

It's redundant

Comment (dmkofgl) [reply src/main/java/dl/news/portal/web/config/Application.java]: I did use it because this class placed in another branch of package hierarchy.

Comment (avinahradau) [reply src/main/java/dl/news/portal/web/config/Application.java]: Change you package hierarchy to the correct form and remove this annotations

Comment (dmkofgl) [reply src/main/java/dl/news/portal/web/config/Application.java]: Done

File: src/main/java/dl/news/portal/domain/entity/News.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.Date;
+
+@Entity
+public class News implements Serializable {
```
Comment (avinahradau) [inline src/main/java/dl/news/portal/domain/entity/News.java]: Do you really need this Serializable? I think not

File: src/test/java/dl/news/portal/domain/repository/NewsRepositoryTest.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
+package dl.news.portal.domain.repository;
```
Comment (avinahradau) [inline src/test/java/dl/news/portal/domain/repository/NewsRepositoryTest.java]: Don't test repositories, test services that use them. You should add services

File: src/test/java/dl/news/portal/domain/repository/UserRepositoryTest.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
+package dl.news.portal.domain.repository;
```
Comment (avinahradau) [inline src/test/java/dl/news/portal/domain/repository/UserRepositoryTest.java]: As I mentioned above, test services

Comment (dmkofgl) [reply src/test/java/dl/news/portal/domain/repository/UserRepositoryTest.java]: Done

File: src/test/java/dl/news/portal/domain/repository/NewsRepositoryTest.java (hunk 3 above)
Comment (dmkofgl) [reply src/test/java/dl/news/portal/domain/repository/NewsRepositoryTest.java]: Done

File: src/test/java/dl/news/portal/domain/repository/NewsServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+import javax.validation.ConstraintViolationException;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("test")
```
Comment (avinahradau) [inline src/test/java/dl/news/portal/domain/repository/NewsServiceTest.java]: You dont need this annotation. This env here by default

**Review by avinahradau — APPROVED** [review]

**Pull request merged**